CommandDotNet/AppRunner.cs

```diff
@@ -151,5 +152,11 @@ public AppRunner<T> WithCommandInvoker(Func<ICommandInvoker, ICommandInvoker> co
             _settings.CommandInvoker = commandInvokerProvider(_settings.CommandInvoker);
             return this;
         }
+        
+        public AppRunner<T> UseDependencyResolver(IDependencyResolver dependencyResolver)
+        {
+            DependencyResolver.HostProvidedResolver = dependencyResolver;
+            return this;
```

Comment:
moved this method to this class or access to the private member.  it seemed to fit with the two "With..." methods.